tests/src/tests_tracing.rs

```diff
@@ -41,7 +42,7 @@ pub mod foo {
 #[test]
 #[timed::timed(tracing = true)]
 fn test_tracing() {
-    timed::init_tracing!("Test");
+    timed::init_tracing!("Test", TracingStats::Statistics);
```

Comment:
Can you please add a separate test for this? I would like to keep this one for a single feature.